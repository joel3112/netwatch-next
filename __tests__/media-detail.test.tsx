import { render } from '@testing-library/react';
import MediaDetailPage from '@/pages/[mediaType]/[mediaId]';

describe('Tests movie detail page', () => {
  test('renders page correctly', () => {
    const { container } = render(<MediaDetailPage />);

    expect(container).toMatchSnapshot();
  });
});

describe('Tests tv detail page', () => {
  test('renders page correctly', () => {
    const { container } = render(<MediaDetailPage />);

    expect(container).toMatchSnapshot();
  });
});
