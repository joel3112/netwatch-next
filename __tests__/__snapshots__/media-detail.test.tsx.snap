// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests movie detail page renders page correctly 1`] = `
<div>
  <div
    class="wrapper hasMargins"
  />
</div>
`;

exports[`Tests tv detail page renders page correctly 1`] = `
<div>
  <div
    class="wrapper hasMargins"
  />
</div>
`;
